frontend/packages/ceph-storage-plugin/src/constants/tooltips.ts

```diff
@@ -16,13 +16,15 @@ export const arbiterText = (t: TFunction) =>
   t(
     'ceph-storage-plugin~If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.',
   );
-export const attachDevices = (t: TFunction) =>
+export const attachDevices = (t: TFunction, scName: string) =>
   t(
-    'ceph-storage-plugin~Selected nodes are based on the selected storage class. The selected nodes will preferably be in 3 different zones with a recommended requirement of 14 CPUs and 34 GiB per node.',
+    'ceph-storage-plugin~Selected nodes are based on the storage class {{storageClassName}} and with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    { storageClassName: scName },
   );
-export const attachDevicesWithArbiter = (t: TFunction) =>
+export const attachDevicesWithArbiter = (t: TFunction, scName: string) =>
   t(
-    'ceph-storage-plugin~Selected nodes are based on the selected storage class. 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    'ceph-storage-plugin~Selected nodes are based on the storage class {{storageClassName}} and fulfill the stretch cluster requirements with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    { storageClassName: scName },
```

Comment:
If you are getting missing i18n key errors in console logs then use `<Trans/>`
```suggestion
    'ceph-storage-plugin~Selected nodes are based on the storage class <em>{{storageClassName}}</em> and fulfill the stretch cluster requirements with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
    { storageClassName: scName },
```